### PR TITLE
UX-969, UX-970/RBAC Dashboard & Deploy, Drift Coming Soon page

### PR DESCRIPTION
**Pull request by kevinylee1991**

### CODE REVIEW

File: src/app/core/components/CRUDListContainer.js [hunk 1]
```diff
@@ -64,6 +64,8 @@ const CRUDListContainer = ({
 
   const handleDeleteConfirm = useCallback(async () => {
     toggleConfirmDialog()
+    console.log('handling delete confirm')
```
Comment (jruddell) [inline src/app/core/components/CRUDListContainer.js:67]: console.log

File: src/app/core/components/wizard/Wizard.js [hunk 2]
```diff
@@ -152,6 +152,7 @@ class Wizard extends PureComponent {
 
   render() {
     const { wizardContext, setWizardContext, steps, activeStep } = this.state
+    console.log(wizardContext)
```
Comment (jruddell) [inline src/app/core/components/wizard/Wizard.js:155]: console.log

File: src/app/core/utils/routes.ts [hunk 3]
```diff
@@ -882,6 +882,50 @@ export const routes = {
       },
       name: 'RBAC:ClusterRoleBindings:Edit',
     }),
+    profiles: {
+      list: Route.register({
+        url: `${pluginRoutePrefix}/rbac_profiles`,
```
Comment (jruddell) [inline src/app/core/utils/routes.ts:887]: lets use `/rbac/profiles`

File: src/app/plugins/kubernetes/components/rbac/profiles/actions.ts [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    return response
+  },
+  createFn: async (data) => {
+    Bugsnag.leaveBreadcrumb('Attempting to create rbac profile')
+  },
+  updateFn: async (data) => {
+    Bugsnag.leaveBreadcrumb('Attempting to update rbac profile')
+  },
+  deleteFn: async ({ name }, currentItems) => {
+    Bugsnag.leaveBreadcrumb('Attempting to update rbac profile')
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/rbac/profiles/actions.ts:25]: make sure to use analytics tracking or a new helper method that will do both

File: src/app/plugins/kubernetes/components/rbac/profiles/actions.ts [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      metadata: {
+        name: `${profileName}-${clusterName}-${uuid.v4()}`,
+      },
+      spec: {
+        clusterRef: clusterName,
+        profileRef: profileName,
+        dryRun: false,
+      },
+    }
+    Bugsnag.leaveBreadcrumb('Attempting to create rbac profile binding')
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/rbac/profiles/actions.ts:54]: analytics.track

File: src/app/plugins/kubernetes/components/rbac/profiles/rbac-profiles-index-page.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+import Tabs from 'core/components/tabs/Tabs'
+import Tab from 'core/components/tabs/Tab'
+import RbacProfilesListPage from './rbac-profiles-list-page'
+import PageContainer from 'core/components/pageContainer/PageContainer'
+import DriftAnalyticsPage from './drift-analytics-page'
+// Need to import the actions file somewhere in the app for it
+// to load up, is there somewhere better?
+//@ts-ignore
+import { rbacProfilesActions } from './actions'
+console.log(rbacProfilesActions, 'rbac profiles actions')
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/rbac/profiles/rbac-profiles-index-page.tsx:11]: console.log... also what is the issue you're having here?

File: src/app/plugins/kubernetes/components/rbac/profiles/rbac-profiles-list-page.tsx [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+import ProfilePublishDialog from './profile-publish-dialog'
+import SimpleLink from 'core/components/SimpleLink'
+
+// const defaultParams = {
+//   orderBy: 'name',
+//   orderDirection: 'asc',
+// }
+// const usePrefParams = createUsePrefParamsHook('RbacProfiles', listTablePrefs)
+
+// const ListPage = ({ ListContainer }) => {
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/rbac/profiles/rbac-profiles-list-page.tsx:24]: is this commented out code still needed?

File: src/app/plugins/kubernetes/index.js [hunk 8]
```diff
@@ -392,6 +393,18 @@ Kubernetes.registerPlugin = (pluginManager) => {
       link: { path: '/alarms', exact: true },
       component: MonitoringPage,
     },
+    {
+      name: 'RBAC Profiles',
+      requiredRoles: 'admin',
+      link: { path: '/rbac_profiles', exact: true },
```
Comment (jruddell) [inline src/app/plugins/kubernetes/index.js:399]: /rbac/profiles

**Review by jruddell — APPROVED** [review]

File: src/app/core/utils/routes.ts (hunk 3 above)
Comment (kevinylee1991) [reply src/app/core/utils/routes.ts:887]: using /rbac/profiles as the path causes rbac tab to be considered active when on the rbac profiles tab, so I changed this to rbac_profiles

File: src/app/plugins/kubernetes/components/rbac/profiles/rbac-profiles-index-page.tsx (hunk 6 above)
Comment (kevinylee1991) [reply src/app/plugins/kubernetes/components/rbac/profiles/rbac-profiles-index-page.tsx:11]: oops this was an issue I had at the beginning when I didn't have the actions file imported anywhere, it basically just didn't get registered so when the list table tried to query it using the ActionDataKey, the app crashed. I can remove this part completely

File: src/app/plugins/kubernetes/components/rbac/profiles/rbac-profiles-list-page.tsx (hunk 7 above)
Comment (kevinylee1991) [reply src/app/plugins/kubernetes/components/rbac/profiles/rbac-profiles-list-page.tsx:24]: luckily not needed anymore after they made all rbac profiles be under the same namespace

**Review by xantorres — APPROVED** [review]

**Pull request merged**